modules/grapher.js

```diff
@@ -366,6 +367,20 @@ Grapher.prototype.transform = function (transform) {
 };
 
 /**
+  * grapher.nodeScale
+  * ------------------
+  *
+  * Set the nodeScale. The displayed radius of the node will be the nodeScale multiplied by the node radius.
+  * If no arguments are passed in, returns the current scale.
+  */
+Grapher.prototype.nodeScale = function (scale) {
+  if (u.isUndefined(scale)) return this.props.nodeScale;
+  if (u.isNumber(scale)) this.props.nodeScale = scale;
```

Comment:
typeof NaN === 'number' if we want to double check that scale is not a NaN, there needs to be additional check.  But this is not a big deal anyways.